starport_template/lib/starport_app.dart

```diff
@@ -34,6 +34,7 @@ class StarportApp extends StatelessWidget {
         brightness: themeStore.isDarkTheme ? Brightness.dark : Brightness.light,
         child: Builder(
           builder: (context) {
+            setErrorBuilder(context);
```

Comment:
not really sure why we're setting a variable as part of the build method, and why this variable is static? Looks like flutter team at google was a bit drunk when implemented this O.o. :D Wouldn't it make more sense to have specialized widget that is responsible of building the Error UI ? I think `ErrorWidget` is more meant for general build errors and altering it's `builder` field you're updating the way all build errors are being displayed across the whole app.

Reply:
https://docs.flutter.dev/testing/errors, yes they were drunk, i figured it would be great if we had our own custom error screen app-wide, gets rid of the red screen of death and allows us to still copy error messages with one click of a buttion